discord/http.py

```diff
@@ -217,7 +217,7 @@ def __init__(
         self.proxy: str | None = proxy
         self.proxy_auth: aiohttp.BasicAuth | None = proxy_auth
         self.use_clock: bool = not unsync_clock
-        self._executors = []
+        self._executors: list[Executor] = []
 
         user_agent = (
             "DiscordBot (https://pycord.dev, {0}) Python/{1[0]}.{1[1]} aiohttp/{2}"
```

Comment:
Would be cool if we could make it so a user can change this in some way or another